### DISCUSSION TITLE
How to keep tag attributes when replacing a tag surround

### DISCUSSION THREAD

**Discussion by mikaoelitiana** [nvim-mini/mini.nvim#458] [General]
Let's say I have a tag like this : `<span style="some styles...">some content</span>`. If I replace the `span` with a `p` using mini.surround, the whole tag with it's `style` attribute is replaced and I end up with `<p>some content</p>`. 

Is there a way to keep the tag attributes and have something like `<p style="some styles...">some content</p>` at the end?

**Comment by echasnovski** [comment]
Not really possible to "preserve" attributes per se. See #397.
